dockerfiles/README.md

```diff
@@ -0,0 +1,54 @@
+# System Docker Compose
+
+This docker-compose file is meant to provide an easy way to setup the vulcanize system without requiring building any custom code. This setup provides:
+
+* A Geth Node with state-diffs emitted
+* A header-sync service
+* An extract-diffs service with maker storage transformers.
+* An execute service with the default maker transforms [mcd-transformers](https://github.com/makerdao/vdb-mcd-transformers).
+* An execute service with the oasis transforms [oasis](https://github.com/makerdao/vdb-oasis-transformers).
+* A postgraphile instance for API based queries of the data [postgraphile](https://github.com/makerdao/vdb-postgraphile).
+* And finally a postgres database.
+
+Running all of this via docker-compose will give you a very similar setup to the public api.
+
+
+## Dependencies
+ - Docker (the rest is handled by docker-compose)
+
+## System Startup
+
+You can start the system by running 
+
+```bash
+docker-compose -f dockerfiles/docker-compose.yml up
+
+```
+
+This will work but keep in mind the provided geth-statediffing image will need to sync from the beginning of the chain, and at the time of this writing the provided header-sync will not start updating until block number 71783773. At that point it will begin syncing headers to your database on each block, and will not start syncing storage diffs until some time after that. This means it will need to run for a long time before being viable.
```

Comment:
If you wanted to add detail, you could mention that it will start syncing storage diffs from the earliest block of a contract configured in the `extractDiffs.toml` config currently used by the extract-diffs image, which is currently block 8925057 (MEDIAN_ETH_1_0_0 deployment block)